src/clients/ClientSettings.tsx

```diff
@@ -47,6 +50,9 @@ export const ClientSettings = ({
     "attributes.display.on.consent.screen"
   );
   const protocol = watch("protocol");
+  const idpInitiatedSsoUrlName: string = watch(
```

Comment:
I think you can pass a generic here instead.
```suggestion
  const idpInitiatedSsoUrlName = watch<string>(
```